### ISSUE TITLE
Enhancement Request: Improve Accessibility Features

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#34]
Hi team,

I wanted to kindly point out that the current version of the studygroupmap app seems to be lacking some accessibility features. It would be great if we could enhance the accessibility to make sure everyone can use the app comfortably and efficiently. Thanks for considering this improvement!

Best regards.
